pkg/utils/utils.go

```diff
@@ -147,14 +147,16 @@ func RandomSeq(n int) string {
 var informer sync.Once
 var informerInstance *Informers
 
+const informerDefaultResync = 12 * time.Hour
+
 func instantiateInformers(kubeClient KubeClientIntf, registeredInformers []string, ocs oshiftclientset.Interface, namespace string) *Informers {
 	cs := kubeClient.ClientSet
 	var kubeInformerFactory kubeinformers.SharedInformerFactory
 	if namespace == "" {
-		kubeInformerFactory = kubeinformers.NewSharedInformerFactoryWithOptions(cs, time.Second*30)
+		kubeInformerFactory = kubeinformers.NewSharedInformerFactoryWithOptions(cs, informerDefaultResync)
```

Comment:
Can we please take the resync time as an argument to `instantiateInformers`? If no argument is provided, then we can take the default value of `informerDefaultResync`?

Reply:
Done.